README.md

```diff
@@ -51,7 +53,7 @@ $ colcon build --packages-up-to soss-dds
 
 6. In the workspace folder, execute colcon:
     ```
-    $ colcon build --packages-up-to soss-dds soss-ros2
+    $ colcon build --packages-up-to soss-dds soss-ros2-test
```

Comment:
Most users probably aren't interested in run the tests. Maybe we should add this possibility as a note.

Reply:
This addition is because ros2 only generate the empy files for the types if the tests are enabled.

Reply:
Then I insist on the note.